lib/view/students/students_screen.dart

```diff
@@ -64,17 +64,25 @@ class StudentsPage extends StatelessWidget {
         controllerOfDescription: TextEditingController(text: ""),
         birthdate: "Select Date",
         controllerOfHours: TextEditingController(text: ""));
-        // -------------------------------------->
+    // -------------------------------------->
   }
 
-  InkWell studentItem(
-      Students studentInfo, int index, BuildContext context) {
+  InkWell studentItem(Students studentInfo, int index, BuildContext context) {
     return InkWell(
       onTap: () => onTapItem(context, studentInfo),
-      child: PrimaryContainerStudents(index: index, modelStudents: studentInfo),
+      child: primaryContainer(index, studentInfo),
     );
   }
 
+  PrimaryContainer primaryContainer(int index, Students studentInfo) {
```

Comment:
Nice idea, but ...
Is this primary container "primary" for the application or just for the student?
Better call it something like "StudentContainer" or at least make it private.
Primary gives the meaning that it is used by the entire application (as a component).

Reply:
solved